docs/book/user-guide/advanced-guide/client.md

```diff
@@ -0,0 +1,180 @@
+artifact stores as part of pipeline runs.
+- **Schedules**: Metadata about the schedules that you have used to 
+[schedule pipeline runs](schedule-pipeline-runs).
+- **Builds**: The pipeline-specific Docker images that were created when
+[containerizing your pipeline](containerize-your-pipeline.md).
+- **Code Repositories**: The git code repositories that you have connected with
+your ZenML instance. See [here](connect-your-git-repository) for more information.
+
+{% hint style="info" %}
+Checkout the [documentation on fetching runs](../starter-guide/fetch-runs-after-execution.md)
+for more information on the various ways how you can fetch and use the
+pipeline, pipeline run, step run, and artifact resources in code.
+{% endhint %}
+
+### Stacks, Infrastructure, Authentication
+- **Stack**: The stacks registered in your ZenML instance.
+- **Stack Components**: The stack components registered in your ZenML instance,
+e.g., all orchestrators, artifact stores, model deployers, ...
+- **Flavors**: The [stack component flavors](../../getting-started/core-concepts.md#flavor)
+available to you, including:
+    - Built-in flavors like the [local orchestrator](../component-guide/orchestrators/local.md),
+    - Integration-enabled flavors like the [Kubeflow orchestrator](../component-guide/orchestrators/kubeflow.md),
+    - Custom flavors that you have [created yourself](../../platform-guide/set-up-your-mlops-platform/implement-a-custom-integration#step-2-create-individual-stack-component-flavors). 
+- **User**: The users registered in your ZenML instance. If you are running
+locally, there will only be a single `default` user. For more information on
+user management see [here](../../platform-guide/set-up-your-mlops-platform/user-management.md).
+- **Secrets**: The infrastructure authentication secrets that you have 
+registered in the 
+[ZenML Secret Store](../../platform-guide/set-up-your-mlops-platform/use-the-secret-store/use-the-secret-store.md).
+- **Service Connectors**: The service connectors that that you have set up to
+[connect ZenML to your infrastructure](../../platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure.md).
+
+## Client Methods
+
+### CRUD (Create, Read, Update, Delete)
+
+For most resources, the ZenML Client provides five different CRUD methods:
+
+#### List
+
+Get a list of resources, e.g.:
+
+```python
+client.list_pipeline_runs(
+    stack_id=client.active_stack_model.id,  # filter by stack
+    user_id=client.active_user.id,  # filter by user
+    sort_by="desc:start_time",  # sort by start time descending
+    size=10,  # limit page size to 10
+)
+```
+
+These methods always return a 
+[Page](https://sdkdocs.zenml.io/0.41.0/core_code_docs/core-models/#zenml.models.page_model)
+of resources that contains, by default, the first 50 results. You can modify the
+page size by passing the `size` argument or fetch a subsequent page by passing 
+the `page` argument to the list method.
```

Comment:
Maybe worth mentioning that `Page` objects are subclasses of `List`s? Makes them a bit more approachable?

Reply:
Actually they are pydantic model subclasses which we modified to behave like lists ;)

Reply:
but yeah will add something that says "this is basically a list"